http/src/main/resources/reference.conf

```diff
@@ -209,11 +209,19 @@ azure {
    coa-app-config {
      # TODO (TOAZ-243): Add chart to Dockerfile once it's published
      chart-name = "/Users/rtitle/git/broadinstitute/cromwhelm/coa-helm"
```

Comment:
Just noting TODO: we should move to a published version of the coa-helm chart soon